scripts/testScripts.sh

```diff
@@ -0,0 +1,115 @@
+#!/bin/bash
```

Comment:
I believe this is a utility script. it should not be checked in.

Reply:
Yup yup - I made a commit to delete it but forgot to push it up - it should be deleted now